src/buffer/out/textBuffer.cpp

```diff
@@ -2207,3 +2208,34 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
 
     return hr;
 }
+
+// Method Description:
+// - Adds a hyperlink to our hyperlink table
+// Arguments:
+// - The hyperlink URI
+void TextBuffer::AddHyperlinkToMap(std::wstring uri) noexcept
+{
+    _hyperlinkMap.insert(std::pair<SHORT, std::wstring>(_curHyperlinkId, uri));
+    _curHyperlinkId++;
+}
+
+// Method Description:
+// - Retrieves the next hyperlink ID to use - this is needed for setting
+//   the hyperlink ID in text attributes
+// Return Value:
+// - the SHORT hyperlink ID
+SHORT TextBuffer::GetCurHyperlinkId() const noexcept

```

Comment:
I wouldn't use the abbreviation 'Cur' here. I'd write `GetNextHyperlinkId` or `GetCurrentHyperlinkId`. 

We have Cursors all over the place which could be shortened to `Cur` too.

Reply:
Fixed!